arches/app/utils/i18n.py

```diff
@@ -101,6 +101,38 @@ def get_localized_value(obj, lang=None, return_lang=False):
         return {found_lang: obj[found_lang]} if return_lang else obj[found_lang]
 
 
+def rank_label(value, *, kind='prefLabel', label_lang, sought_lang=None):
```

Comment:
this pattern of forcing named parameters (using the *) has never been used in Arches.  It took me a while to even find out what this meant.  I would suggest that we remove this and then give `label_lang` a default value of `None`

Reply:
No problem!

Just to add context for what motivated it, what I like about keyword-only params is that they help guard against confusing very similar arguments for each other. ("What's a label_lang -- the language _this_ label is in or the language this label _should_ be in?") Forcing someone to type out the param name hopefully helps readers/writers of the code reinforce which argument value belongs to which argument, but yeah, just a matter of style, and if we don't use `*` anywhere yet, this isn't the place to do it :-)

Reply:
I could probably find a better param name than `label_lang`!

Reply:
My understanding of `label_lang` is that we'll always have it, and defaulting to None and guessing English would be less ideal. That is, for "projet" we'll never be in doubt that it's `'fr`'--but let me know if I'm wrong about that.

Reply:
Please forgive my waffling--it's just a rank function, so we can be permissive and then just rank it low if the language of the label is unknown 👍

Reply:
Updated in [dd84ea7](https://github.com/archesproject/arches/pull/9849/commits/dd84ea794f518db63154519f29c909765cc35e87)